source/interface/directoryContents.js

```diff
@@ -13,7 +13,7 @@ function getDirectoryContents(remotePath, options) {
         method: "PROPFIND",
         headers: {
             Accept: "text/plain",
-            Depth: 1
+            Depth: options.depth || 1
```

Comment:
This can be removed as it's part of another issue - #130

Reply:
Got it